canonical-web-and-design/ubuntu.com
# Add recaptcha script to /internet-of-things/edgex and /whitepapers/beta/robotics

## Done

- Add recaptcha script to /internet-of-things/edgex and /whitepapers/beta/robotics

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/edgex and http://0.0.0.0:8001/whitepapers/beta/robotics
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the recaptcha form appears and the form submits


## Issue / Card

Fixes #9205
